leshan-server-demo/webapp/src/js/valueutils.js

```diff
@@ -22,6 +22,9 @@ function valueToString(value, type) {
   } else if (type == "time") {
     let date = new Date(value);
     return date.toDateString()+ " - " +  date.toLocaleTimeString() + " (" + date.getTime()/1000 + "s)";
+  } else if (type == "corelink") {
+    // for now display the corelink as a json structure
+    return JSON.stringify(value);
```

Comment:
we should probably implement this ?
Or eventually change the JSON serialization and just return a string for corelink for now.

(As you prefer)

Reply:
I like the fact the link is presented in the API a way you don't have to parse again, but I have no idea how to do a clean presentation in the front code

Reply:
> I like the fact the link is presented in the API a way you don't have to parse again,

Yep but as we don't use it in the UI this could be OK to have value as string in the UI waiting we need something better.

>  I have no idea how to do a clean presentation in the front code

I think the classic string representation of Link is OK. (it should not be so hard to implement it in javascript I guess :thinking: )

and alternative (probably not so good) solution having both representation in the REST API. 
e.g. ` { str : "</3>;</4>...", val: { ... }}`

Reply:
done, I serialize it back to corelnk string and display as-is:
![image](https://user-images.githubusercontent.com/899073/171036132-9752b169-8c4b-40a8-b409-e9107ca3d886.png)